source/download.html

```diff
@@ -76,7 +76,7 @@ <h3 id="struts{{ site.current_version_short }}">Struts {{ site.current_version }
 
 <ul>
   <li>
-    <a href="https://struts.apache.org/docs/version-notes-{{ site.current_version_short }}.html">Version Notes</a>
+    <a href="{{ site.wiki_url }}/Version+Notes+{{ site.current_version }}.html">Version Notes</a>
```

Comment:
Please drop the `.html` suffix as Confluence doesn't support it
https://cwiki.apache.org/confluence/display/WW/Version+Notes+2.5.14.1

Reply:
done with .html